engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/expression/IfThenElse.scala

```diff
@@ -0,0 +1,158 @@
+package org.enso.compiler.core.ir
+package expression
+
+import org.enso.compiler.core.Implicits.{ShowPassData, ToStringHelper}
+import org.enso.compiler.core.{IR, Identifier}
+import org.enso.compiler.core.IR.{indentLevel, mkIndent}
+
+import java.util.UUID
+
+/** The Enso case expression. */
+case class IfThenElse(
+  val cond: Expression,
```

Comment:
nit: `val` modifier is redundant for parameter of case class primary constructor